google/cloud/pubsub/integration_tests/subscription_admin_integration_test.cc

```diff
@@ -58,10 +58,8 @@ TEST(SubscriptionAdminIntegrationTest, SubscriptionCRUD) {
                             pubsub_testing::RandomSubscriptionId(generator));
 
   auto topic_admin = TopicAdminClient(MakeTopicAdminConnection());
-  auto subscription_admin_connection =
-      pubsub::MakeSubscriptionAdminConnection();
   auto subscription_admin =
-      SubscriptionAdminClient(subscription_admin_connection);
+      SubscriptionAdminClient(pubsub::MakeSubscriptionAdminConnection());
```

Comment:
Nit: It is a little jarring to see the `pubsub` qualification here but not elsewhere.

Reply:
Fixed.